docs/_playbooks/trustar_enrich_indicators.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "TruSTAR Enrich Indicators"
+last_modified_at: 2021-11-24
+toc: true
+toc_label: ""
+tags:
+  - Input
+  - Splunk SOAR
+  - TruSTAR
```

Comment:
Could we please add "Investigate" and "Threat Intel" here as tags to be consistent with the platform?

Reply:
I added all of the tags.platform_tags from the playbook YAMLs to populate the tags section of each page.